README.md

```diff
@@ -150,18 +110,59 @@ Create a `Scrabble::TileBag` class with __full unit testing/specs__. You should
 | L : 4  | Y : 2 |
 | M : 2  | Z : 1 |
 
-#### Modifications to `Player`
-Create specs for and add to the `Player` class the following instance methods:
+There is no requirement for the program to change this letter pool over time. Imagine that every time this method is called, it is a new letter pool every time.
+
+### Wave 2
+
+Implement the following method in the file `adagrams.rb`:
+
+- `uses_available_letters? input, letter_bank`
+  - This method should check if the input word only uses characters that are contained within the passed in letter bank. Essentially, this method checks if the word is, indeed, an anagram of some or all of the given letters
+  - Has two parameters:
+     - `input`, the first parameter, describes some input word, and is a string
```

Comment:
Could we change these parameter names to be `word` and `hand`?

What is our official line on using parentheses in argument lists? I think I remember that we encourage it (so this would be `uses_available_letters?(word, hand)`), but that might just be me and not Ada.